cmd/sealos/cmd/delete.go

```diff
@@ -32,9 +32,16 @@ delete to nodes:
 		Use 'sealos add' to recover:
 			sealos add --nodes x.x.x.x
 
-delete to default cluster: 
+delete to masters:
+	sealos delete --masters x.x.x.x
+
+delete to masters and nodes:
 	sealos delete --masters x.x.x.x --nodes x.x.x.x
 	sealos delete --masters x.x.x.x-x.x.x.y --nodes x.x.x.x-x.x.x.y
+
+Please note:
```

Comment:
Please note that sealos will delete your master if the --masters parameter is specified.